Lib/telnetlib.py

```diff
@@ -431,18 +431,20 @@ def process_rawq(self):
         buf = [b'', b'']
         try:
             while self.rawq:
-                c = self.rawq_getchar()
                 if not self.iacseq:
-                    if c == theNULL:
-                        continue
-                    if c == b"\021":
-                        continue
-                    if c != IAC:
-                        buf[self.sb] = buf[self.sb] + c
-                        continue
+                    slice = self._rawq_getslice()
+                    if slice:
+                        buf[self.sb] = buf[self.sb] + slice
                     else:
-                        self.iacseq += c
+                        c = self.rawq_getchar()
+                        if c == theNULL:
```

Comment:
Can `theNull` be replaced with `b'\0'`?

Reply:
I think `theNull` should be replaced by `NOOPT` (they're equal), but it's another topic. This comparison comes from the initial implementation, I've only reindented.